pkgs/development/libraries/onnxruntime/default.nix

```diff
@@ -27,64 +27,67 @@ let
     owner = "HowardHinnant";
     repo = "date";
     rev = "v2.4.1";
-    sha256 = "sha256-BYL7wxsYRI45l8C3VwxYIIocn5TzJnBtU0UZ9pHwwZw=";
+    hash = "sha256-BYL7wxsYRI45l8C3VwxYIIocn5TzJnBtU0UZ9pHwwZw=";
   };
 
   eigen = fetchFromGitLab {
     owner = "libeigen";
     repo = "eigen";
-    rev = "d10b27fe37736d2944630ecd7557cefa95cf87c9";
-    sha256 = "sha256-Lmco0s9gIm9sIw7lCr5Iewye3RmrHEE4HLfyzRkQCm0=";
+    rev = "e7248b26a1ed53fa030c5c459f7ea095dfd276ac";
+    hash = "sha256-uQ1YYV3ojbMVfHdqjXRyUymRPjJZV3WHT36PTxPRius=";
```

Comment:
It may seem out of scope, but can we please try remove this and patch the source to use `FIND_PACKAGE_ARGS` instead